pymeasure/adapters/ni_gpib_232ct.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, resource_name, address=None, serial_timeout=1000, rw_delay=50
             self.eoi = eoi
         self.flush_read_buffer()
 
-    class GPIBStatus(IntFlag):
+    class GPIBStatus(int, Flag):
```

Comment:
don't do that yet, we are not yet clear if this is the good way to fix this python 3.11 problem (#896)

Reply:
> don't do that yet, we are not yet clear if this is the good way to fix this python 3.11 problem (#896)

Thanks for the wrning, reverted the last commit